tests/SystemTests.py

```diff
@@ -0,0 +1,91 @@
+from unittest import TestCase,main
+import mock
+
+from hsl_builder import System,SystemEvents
+
+class SystemCompleteTest(TestCase):
+
+    def setUp(self):
+        self.system = System("complete", SystemEvents.COMPLETE)
+
+    def test_initialization(self):
+        """
+        verify that System Message  is initialized with the correct options
+        """
+        self.assertEqual(self.system.text, "complete")
+        self.assertEqual(self.system.event, SystemEvents.COMPLETE)
+        self.assertEqual(self.system.type, "SYSTEM")
+
+    def test_hsl_default(self):
+        """
+        verify the default hsl generated
+        """
+        self.assertDictEqual(self.system.to_hsl(),{
+            'text': 'complete',
+            'type': 'SYSTEM',
+            'voice_text': '',
+            'data': {
+                'event_name': 'chat_complete',
+                'payload': {}
+            }
+        })
+
+    def test_hsl_payload(self):
+        """
+        verify that payload is added in the generated hsl
+        """
+        self.system.payload = {"key": "value"}
+        self.assertDictEqual(self.system.to_hsl(),{
+            'text': 'complete',
+            'type': 'SYSTEM',
+            'voice_text': '',
+            'data': {
+                'event_name': 'chat_complete',
+                'payload': {"key": "value"}
```

Comment:
This event has details of agent when it comes. Can we please add it in test case?